### PR TITLE
Assignment 1

### PR DESCRIPTION
**Pull request by aserdobintsev**

### CODE REVIEW

**Comment by aserdobintsev** [comment]
@mellonis did configure TravisCI for this repo?

**Comment by mellonis** [comment]
> @mellonis did you configure TravisCI for this repo?

I need to add a repository for ci on travis-ci.org manually. This repo was created after your assignment was accepted. So, I have done it only now.

File: assignment.js [hunk 1]
```diff
@@ -2,5 +2,7 @@
 // prototype: bind(func, thisArg[, arg1[, arg2[, ...]]])
 
 export default bind(f, ctx, ...bindArgs) {
```
Comment (mellonis) [inline assignment.js]: when you write func(), you call the func function
when you write function func(), you define a function named func

Comment (aserdobintsev) [reply assignment.js]: You are right.
I've misunderstood the syntax for default export in docs [here](https://developer.mozilla.org/en-US/docs/web/javascript/reference/statements/export)

**Pull request merged**